packages/connectivity/src/scp-cf/jwt.ts

```diff
@@ -32,12 +32,13 @@ export function userId({ user_id }: JwtPayload): string {
 }
 
 /**
- * @internal
  * Get the tenant ID of a decoded JWT, based on its `zid` or if not available `app_tid` property.
  * @param jwtPayload - Token payload to read the tenant ID from.
+ * @param jwtPayload.zid - The zone ID, representing the tenant in the XSUAA token.
+ * @param jwtPayload.app_tid - The app tenant ID, representing the tenant in the IAS token.
```

Comment:
[q] Have you checked how the docs are rendered with this? I think this doesn't work and maybe it is not needed (because you are describing what is in the `JwtPayload` and it is pointed out in the docs above.

Reply:
Mostly added it because vale is crying otherwise, I guess I can also exclude this instead.